src/Message/PurchaseRequest.php

```diff
@@ -70,11 +70,14 @@ private function sign($params, $api_key) {
 
 	public function sendData($data)
     {
+		// FIXME: You're not actually sending any data here.
```

Comment:
@judgej Thanks for your explanation. In the case of the action (send) that triggers the redirect, it should be fine to just create the response with the collected data, right? As it is implemented right now?
In other words: I send the action "purchase" to OmniPay. And the response is a redirect…